Justfile

```diff
@@ -0,0 +1,21 @@
+alias c := check
+alias d := doc
+alias f := format
+alias fmt := format
+
+default:
+    just --list
+
+ci: check doc format
+
+check:
+    cargo clippy --target wasm32-unknown-unknown -- -Dwarnings
```

Comment:
since its supposed to be a no-op on other platforms lets at least build one non-wasm32 target here too